cli/deployments/registry.go

```diff
@@ -57,12 +57,26 @@ func (k Registry) Describe() string {
 func (k Registry) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	ui.Note().Msg("Removing Registry...")
 
+	message := "Deleting Registry namespace " + RegistryDeploymentID
+	warning, err := helpers.WaitForCommandCompletion(ui, message,
+		func() (string, error) {
+			return c.DeleteNamespaceIfOwned(RegistryDeploymentID)
+		},
+	)
+	if err != nil {
+		return errors.Wrapf(err, "Failed deleting namespace %s", RegistryDeploymentID)
+	}
+	if warning != "" {
+		ui.Exclamation().Msg(warning)
+		return nil
```

Comment:
Same as above

Reply:
Added a comment and exiting only for a specific warning.